### ISSUE TITLE
Screen Reader: Chat: Informative icons in user inbox not announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74851]
## Action Performed
Prerequisite:
User has an account and is logged in.

1. Using Windows Chrome + JAWS, open the page: [https://new.expensify.com/\](https://new.expensify.com//)
2. In the Inbox page, tab to focus any conversation under 'Inbox'.
3. Note what is announced by the screen reader.

## Expected Result
The screen reader should announce the presence and meaning of the icons such as pinned or green color icon. 

## Actual Result
When focus moves to a conversation containing these icons pinned or green circle, the screen reader only announces the conversation’s sender and message preview. It does not announce the additional visual indicators, such as the green dot showing online presence or the pin icon indicating a pinned conversation. As a result, screen reader users receive less contextual information than sighted users and cannot determine whether a conversation is pinned or if the sender is currently online.

HTML:
<button aria-label="Navigates to a chat Concierge. It's great to meet you!" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1wtj0ep r-1awozwy r-1xfd6ze r-18u37iz r-jusfrs r-1m04atk r-1pyaxff r-rd2f51 r-13wfysu r-9ffhgg r-lrvibr" data-testid="811304579477876" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx" style=""><div class="css-g5y9jx" style="min-width: 0px; text-decoration-line: none; flex: 1 1 0%; min-height: 64px; padding-top: 12px; padding-bottom: 12px; justify-content: center;"><div class="css-g5y9jx r-18u37iz r-1awozwy"><div class="css-g5y9jx"><div class="css-g5y9jx r-y47klf r-1b7u577 r-633pao" data-testid="ReportActionAvatars-SingleAvatar"><div class="css-g5y9jx"><div class="css-g5y9jx" data-testid="SvgFallbackAvatar Icon" style="overflow: hidden; border-radius: 100px;"><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" id="concierge-avatar_svg__Layer_1" x="0" y="0" viewBox="0 0 80 80" width="40" height="40" hovered="false" pressed="false"><style>.concierge-

## Area issue was found
Chat

## Failed Checkpoint
WCAG 1.1.1 Non-Text Content
https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html

## User Impact
Users who have difficulty perceiving visual content use assistive technology to read text aloud, present it visually, or convert it to braille. Text alternatives may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. People who are deaf-blind can read the text alternatives in braille.

## Suggested Resolution
Ensure the screen readers announce the icon status when a conversation is focused. This can be done by providing role="img" to the svg image and providing a descriptive name to it by using "aria-label" attribute.

Refer:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/img_role

## Environment(s)
Windows 10 - Chrome

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/5dbdd0db-f961-44ee-bbcb-f5ea11fd4bd8)

https://github.com/user-attachments/assets/a188e6c8-f324-4276-b14b-8197d01dfb58

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995587
**Issue reported by:** NA 
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
`Accessibility` issue, will be addressed soon.

**Comment by mallenexpensify** [comment]
Accessibility, unassigning, will be addressed in bulk once all issues are created.
